ts/utils/util.py

```diff
@@ -22,51 +23,65 @@ def list_classes_from_module(module, parent_class=None):
     """
 
     # Parsing the module to get all defined classes
-    classes = [cls[1] for cls in inspect.getmembers(module, lambda member: inspect.isclass(member) and
-                                                    member.__module__ == module.__name__)]
+    classes = [
+        cls[1]
+        for cls in inspect.getmembers(
+            module,
+            lambda member: inspect.isclass(member)
+            and member.__module__ == module.__name__,
+        )
+    ]
     # filter classes that is subclass of parent_class
     if parent_class is not None:
         return [c for c in classes if issubclass(c, parent_class)]
 
     return classes
 
+
 def load_label_mapping(mapping_file_path):
     """
     Load a JSON mapping { class ID -> friendly class name }.
     Used in BaseHandler.
     """
     if not os.path.isfile(mapping_file_path):
-        logger.warning('Missing the index_to_name.json file. Inference output will not include class name.')
+        logger.warning(
+            "'index_to_name.json' file is missing. Inference output will not include class name."
```

Comment:
we can improve this to show the mapping file path

Reply:
done